src/App.js

```diff
@@ -112,11 +113,9 @@ function verifySolution() {
 
   for (var index = 0; index < 9; index++) {
     let verifyColBox = verifyCol(index, colSet, rowSet);
-    let verifyRowBox = verifyRow(index, rowSet);
-    console.log("index: " + index + ", verifyRowBox: " + verifyRowBox);
-    
+    let verifyRowBox = verifyRow(index, rowSet);    
```

Comment:
im seeing some of the same changes in both PRs. Typically we don't want to do that. The PRs could conflict with each other.

Reply:
Sorry, they should be put in the same pull request